packages/datadog_flutter_plugin/android/src/test/kotlin/com/datadoghq/flutter/DatadogRumPluginTest.kt

```diff
@@ -190,6 +192,12 @@ class DatadogRumPluginTest {
         val featureConfiguration: Any = config.getFieldValue("featureConfiguration")
         assertThat(featureConfiguration.getPrivate("sampleRate")).isEqualTo(sessionSampleRate)
         assertThat(featureConfiguration.getPrivate("trackFrustrations")).isEqualTo(trackFrustration)
+        if (trackNonFatalAnrs != null) {
+            assertThat(featureConfiguration.getPrivate("trackNonFatalAnrs")).isEqualTo(trackNonFatalAnrs)
+        } else {
+            // If null, default shouldn't be changed. Tests are run on a version that enables ANR tracking by default
+            assertThat(featureConfiguration.getPrivate("trackNonFatalAnrs")).isEqualTo(true)
```

Comment:
Minor API suggestion

```suggestion
            assertThat(featureConfiguration.getPrivate("trackNonFatalAnrs")).isFalse
```

Reply:
Actually it turns out I don't have `isFalse`?

Reply:
ah, maybe `isFalse()` then?

Reply:
Nope (I checked). I don't know why but my library doesn't have anything like `isFalse`